drivers/ws281x/atmega.c

```diff
@@ -190,7 +190,7 @@ void ws281x_write_buffer(ws281x_t *dev, const void *buf, size_t size)
                 /* Total CPU cycles for high period:
                  * 10 cycles for bit 1, 5 cycles for bit 0 */
                 : [data]    "+r" (data),
-                  [port]    "=e" (port_addr)
+                  [port]    "+e" (port_addr)
```

Comment:
sneaky!
But with this it's now working again.

Reply:
Actually it should be read only, not read write. I fixed it in a follow up.

I'm still not that experienced with inline assembly as I would like to be. (It is obviously the memory at the address that is written to, not the register holding that address.)